src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -1020,3 +1020,19 @@ a.tribe-rating-link {
 	top: 6px;
 }
 
+.tribe-message {
+	padding: 1em 0;
+
+	&:before,
+	&:after {
+		content: '';
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+
+	.spirit-animal {
```

Comment:
Should we add a `margin-left` to this sexy beast or rely on the spacing being on the image? My hunch is that you felt the spacing around the image was enough to separate it from the text. If that's good with you, then that's good with me.